### ISSUE TITLE
Uninitialized warnings in tests

### ISSUE THREAD

**Issue by shawnlaffan** [PDLPorters/Alien-Proj4#15]
First reported under https://github.com/PDLPorters/pdl/issues/346

Proj version is 4.9.3, perl version is 5.30.0.  Warnings do not occur with perl 5.28.0.  

Example warnings:

```
Use of uninitialized value $temp in substitution (s///) at C:\shawn\git\Alien-Proj4\blib\lib/Alien/Proj4.pm line 85.
Use of uninitialized value $temp in substitution (s///) at C:\shawn\git\Alien-Proj4\blib\lib/Alien/Proj4.pm line 86.
Use of uninitialized value $temp in split at C:\shawn\git\Alien-Proj4\blib\lib/Alien/Proj4.pm line 87.
```

Looking at the code this might be an issue with reading the projection details in ```load_projection_descriptions```.

All tests pass, FWIW.

**Comment by shawnlaffan** [comment]
Perl -v and gcc versions (both from Strawberry perl on Windows):

```
This is perl 5, version 28, subversion 0 (v5.28.0) built for MSWin32-x64-multi-thread
gcc version 7.1.0 (x86_64-posix-seh, Built by strawberryperl.com project)
```
```
This is perl 5, version 30, subversion 0 (v5.30.0) built for MSWin32-x64-multi-thread
gcc version 8.3.0 (x86_64-posix-seh, Built by strawberryperl.com project)
```


Possibly relevant is that ```invgeod.exe``` and ```invproj.exe``` are generated under perl 5.28 but not 5.30.

**Comment by mohawk2** [comment]
I believe you know a lot more about Proj than I do; can you investigate why those files are generated differently on one version and not the other?

**Comment by shawnlaffan** [comment]
I'll have a look when I get a chance, but if it's in the C bindings then that's on the edges of my skill set.

**Comment by shawnlaffan** [comment]
It looks like the errors are caused by conversion operations being included in the projections list when compiled using Strawberry perl 5.30.  The proj documentation lists these as being in 5.0, but evidently this is not entirely the case.
https://proj.org/operations/conversions/index.html

I am guessing it is triggered by one of the utilities provided with the Strawberry perl PDL version as it does not happen with a non-PDL version of Strawberry perl 5.30.

Given the common use case (as I understand it) the simplest approach is to not add these to the projection information hash. I'll do that now. 

Shawn.

**Comment by mohawk2** [comment]
Thanks! Released as 2.019113.
